recipe/meta.yaml

```
@@ -20,6 +20,7 @@ build:
     - HDF5PLUGIN_AVX512=False
     - HDF5PLUGIN_BMI2=False
     - HDF5PLUGIN_OPENMP=False
+    - HDF5PLUGIN_SYSTEM_LIBRARIES=blosc,blosc2,lz4,zlib,zstd
 
 requirements:
   build:
```

```suggestion
  build:
    - pkg-config   # [unix]
```

Technically it should work on windows to it you use it in your build scripts